crates/db_schema/src/impls/moderator.rs

```diff
@@ -66,6 +66,20 @@ impl Crud for ModRemovePost {
   }
 }
 
+impl ModRemovePost {
+  pub async fn create_multiple(
+    pool: &mut DbPool<'_>,
+    forms: &Vec<ModRemovePostForm>,
+  ) -> Result<usize, Error> {
+    use crate::schema::mod_remove_post::dsl::mod_remove_post;
+    let conn = &mut get_conn(pool).await?;
+    insert_into(mod_remove_post)
+      .values(forms)
+      .execute(conn)
```

Comment:
Diesel can do multiple inserts in the same statement. We should utilize this elsewhere instead of looping SQL commands.

Reply:
Not sure if this is a good solution. If a user with dozens of posts and comments gets banned, this will absolutely spam the mod log of all federated instances.